sklearn/linear_model/ridge.py

```diff
@@ -111,7 +111,7 @@ def _solve_cholesky(X, y, alpha):
         return linalg.solve(A, Xy, sym_pos=True,
```

Comment:
Isn't this handled [here](https://github.com/massich/scikit-learn/blob/is/8769_ridge/sklearn/linear_model/tests/test_ridge.py#L817) If the right test is not passed eventually it would get propagated up and cached there.